Adyen/Core/Components/Installment/InstallmentOptions.swift

```diff
@@ -87,4 +117,40 @@ public struct InstallmentConfiguration {
         self.showInstallmentPrice = false
     }
     
+    @_spi(AdyenInternal)
+    public init(from decoder: Decoder) throws {
+        let container = try decoder.container(keyedBy: DynamicKey.self)
```

Comment:
can you use `decoder .unkeyedContainer()` instead of using the `DynamicKey`?

Reply:
`unkeyedContainer` doesn't have properties like `.allKeys` to compare the key names. Don't think it can be done with it.

Reply:
yeah, I suppose :-)